### ISSUE TITLE
I just tried PWM on the nRF52 and it doesnt seem to work.

### ISSUE THREAD

**Issue by kriswiner** [sandeepmistry/arduino-nRF5#316]
I am using a custom nRF52 development board that works in all respects. Except when I tried to output a PWM signal like:

void setup()
{
pinMode(A2, OUTPUT);
analogWrite(A2, 200);
}

I see zero on A2 (GPIO28).  Looks like PWM is busted for the nRF52 DK variant that I am using for my board variant.

Is this a known problem or is there some magic incantation required to get PWM to work with the current Arduino core?

**Comment by dlabun** [comment]
Yup, It's busted.... Take a look at issue #169 and PR #205 for potential fixes
